public/scripts/experiments/jnd.js

```diff
@@ -524,178 +546,4 @@ class JND {
     hiddenElement.download = this.condition_name + "_jnd_summary_results.csv";
     hiddenElement.click();
   }
-
-
-  /**
-   * Performs the necessary D3 operations to plot distributions depending on graph type.
-   */
-  plot_distributions() {
```

Comment:
@TienSFU25 Did you end up relocating all the plotting code somewhere? Can't find it in your changes anymore?

Reply:
its in JNDTrialDisplay.jsx now